### PR TITLE
Remove enumerize

### PR DESCRIPTION
**Pull request by gmq**
In Platanus we have transitioned to using the standard Rails enums, which means that we are unable to continue supporting enumerize in this gem.

### CODE REVIEW

File: lib/activeadmin_addons/addons/tag_builder.rb [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
     def render
-      @enum_attr = if enumerize_attr?
-                     :enumerize
-                   elsif rails_enum_attr?
+      @enum_attr = if rails_enum_attr?
                      :enum
                    end
 
-      raise "you need to pass an enumerize or enum attribute" unless @enum_attr
+      raise "you need to pass an enum attribute" unless @enum_attr
```
Comment (monkeyci) [inline lib/activeadmin_addons/addons/tag_builder.rb:8]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Add empty line after guard clause.

File: spec/dummy/app/models/invoice.rb [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
   belongs_to :category
   belongs_to :city
   has_and_belongs_to_many :items
   has_and_belongs_to_many :other_items, class_name: 'Item'
 
-  enumerize :state, in: [:pending, :rejected, :approved], default: :pending
+  enum state: {
+    pending: 'pending',
+    rejected: 'rejected',
+    approved: 'approved',
```
Comment (monkeyci) [inline spec/dummy/app/models/invoice.rb:16]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Avoid comma after the last item of a hash.

File: lib/activeadmin_addons/addons/tag_builder.rb [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
```
Comment (difernandez) [inline lib/activeadmin_addons/addons/tag_builder.rb:8]: with the change in `display_data`, `@enum_attr` is no longer used. We could chage this to just:
```ruby
raise "you need to pass an enum attribute" unless rails_enum_attr?
```

File: lib/activeadmin_addons/support/enumerize_formtastic_support.rb
Comment (difernandez) [inline lib/activeadmin_addons/support/enumerize_formtastic_support.rb:1]: this whole file was for enumerize support, we should just delete it

**Review by difernandez — CHANGES_REQUESTED** [review]
Couple small things, looks good!

**Review by difernandez — APPROVED** [review]

**Review by ldlsegovia — APPROVED** [review]

**Pull request merged**